features/step_definitions/helpers.rb

```diff
@@ -79,11 +79,11 @@ def initialize_work_package_params(project, type = Type.find(:first), parent = n
   params
 end
 
-def initialize_impediment_params(project, sprint, user = User.find(:first))
+def initialize_impediment_params(project, sprint, user = User.first)
```

Comment:
Assignment Branch Condition size for initialize_impediment_params is too high. [17.03/15]